classes/class-export.php

```diff
@@ -151,10 +151,9 @@ public function build_record( $item, $columns ) {
 	/**
 	 * Increase pagination limit for CSV Output
 	 *
-	 * @param int $records_per_page Old limit for records_per_page.
 	 * @return int
 	 */
-	public function disable_paginate( $records_per_page ) {
+	public function disable_paginate() {
```

Comment:
Actually this made me wonder why `phpcs` did not catch that issue in the test file. It turns out tests directory is excluded from `phpcs` checks:

https://github.com/xwp/stream/blob/c51860ae8df9c99bdd60891c7b75a0137758a8f2/phpcs.xml.dist#L23

I'm going to see later if it's feasible to enable linter for the test files and if yes, I'm going to issue a separate PR so that it's easier to review.

Reply:
That makes sense! We can do everything there.

Reply:
I've issued a PR which enables a looser `phpcs` linting on the test files. Check it out here: #1509.